README.md

```diff
@@ -92,381 +92,172 @@
-```
 
-If you want to set up testing and various frameworks it\'s probably best
-to check out the [Contributing - Setting
-Up](https://unify.ai/docs/ivy/overview/contributing/setting_up.html#setting-up)
-page, where OS-specific and IDE-specific instructions and video
-tutorials to do so are available!
+    ``` python
+    import ivy
+    import torch
+    import jax
 
-### Using Ivy
+    ivy.set_backend("jax")
 
-You can find quite a lot more examples in the corresponding section
-below, but using Ivy is as simple as:
+    x = jax.numpy.array([1, 2, 3])
+    y = jax.numpy.array([3, 2, 1])
+    z = ivy.add(x, y)
 
-#### Multi-backend Support
+    ivy.set_backend('torch')
 
-``` python
-import ivy
-import torch
-import jax
+    x = torch.tensor([1, 2, 3])
+    y = torch.tensor([3, 2, 1])
+    z = ivy.add(x, y)
+    ```
+    </details>
+1. <details>
+    <summary><b>transpile your code</b></summary>
 
-ivy.set_backend("jax")
 
-x = jax.numpy.array([1, 2, 3])
-y = jax.numpy.array([3, 2, 1])
-z = ivy.add(x, y)
+    ``` python
+    import ivy
+    import torch
+    import jax
 
-ivy.set_backend('torch')
+    def jax_fn(x):
+        a = jax.numpy.dot(x, x)
+        b = jax.numpy.mean(x)
+        return x * a + b
 
-x = torch.tensor([1, 2, 3])
-y = torch.tensor([3, 2, 1])
-z = ivy.add(x, y)
-```
+    jax_x = jax.numpy.array([1, 2, 3])
+    torch_x = torch.tensor([1, 2, 3])
+    torch_fn = ivy.transpile(jax_fn, source="jax", to="torch", args=(jax_x,))
+    ret = torch_fn(torch_x)
+    ```
 
-#### Transpilation API
+    </details>
 
-``` python
-import ivy
-import torch
-import jax
+------------------------------------------------------------------------
 
-def jax_fn(x):
-    a = jax.numpy.dot(x, x)
-    b = jax.numpy.mean(x)
-    return x * a + b
 
-jax_x = jax.numpy.array([1, 2, 3])
-torch_x = torch.tensor([1, 2, 3])
-torch_fn = ivy.transpile(jax_fn, source="jax", to="torch", args=(jax_x,))
-ret = torch_fn(torch_x)
-```
+## 📚 Documentation
 
-## Documentation
+You can find our documentation from this [Ivy Docs page](https://unify.ai/docs/ivy/), which includes:
+- [Motiviation](https://unify.ai/docs/ivy/overview/background.html): This contextualizes the problem Ivy is trying to solve through
+    - the current [ML
+Explosion](https://unify.ai/docs/ivy/overview/background/ml_explosion.html#ml-explosion),
+    - explaining both why is important [to solve this
+problem](https://unify.ai/docs/ivy/overview/background/why_unify.html#why-unify) and
+    - how we adhere to existing [standards](https://unify.ai/docs/ivy/overview/background/standardization.html#standardization) to make this happen.
+- [Related
+Work](https://unify.ai/docs/ivy/overview/related_work.html)   which paints a picture of the role Ivy plays in the ML stack, comparing it to other existing solutions in terms of functionalities and level.
+- [Design](https://unify.ai/docs/ivy/overview/design.html): the design decision, architecture, and building blocks (layers and nodes)  of Ivy.
+- [Deep
```

Comment:
There is no `:` and no space between `Deep dive` and `explains`, also if we capitalize both words in `Related Work` we should do the same in `Deep Dive`.